README.md

```diff
@@ -93,6 +93,22 @@ For example, for `Article` model:
 
 Your WebFlow collection `slug` should be `"articles"`.
 
+If collection `slug` does not match the Rails model collection name, you can still sync with WebFlow collection, but need to specify collection slug for each `CreateItemJob` and `UpdateItemJob` call. If not specified, model name is used as a default slug name.
+
+ 1. `:model_name` - Rails model that has `webflow_site_id` and `webflow_item_id` columns
```

Comment:
This is a string, not a Rails model:

>  1. `model_name` - Rails model name that has `webflow_site_id` and `webflow_item_id` columns

Let's also remove the `:` because these are variables and not ruby symbols.